cpp/src/parquet/properties.h

```diff
@@ -1051,6 +1053,18 @@ class PARQUET_EXPORT ArrowReaderProperties {
   /// Return the Arrow binary type to read BYTE_ARRAY columns as.
   ::arrow::Type::type binary_type() const { return binary_type_; }
 
+  /// \brief Set the Arrow list type to read Parquet list columns as.
+  ///
+  /// Allowed values are Type::LIST and Type::LARGE_LIST.
```

Comment:
Do we need to support list view?

Reply:
We don't *need* to, as list view is probably less useful than binary view.
Moreover, this would need a specific implementation as the current `ListReader` constructs offsets explicitly.